### ISSUE TITLE
Take away universal hull/shield hitpoint knowledge, add it to Tactical Scanner

### ISSUE THREAD

**Issue by bene-dictator** [endless-sky/endless-sky#7890]
### Problem Description

Tactical scanners aren't very useful tactically. The only two uses I can think of are to see whether you have enough crew to take over a ship without having to go board that ship, and whether you spot and overheat a ship running at high heat. Showing the distance value is not all that useful because I'm not going to be able to look at it during battle, and I don't really want to remember weapons distances. 

### Related Issue Links

#5820 - deals with capturing

I don't really know of anything else. 

### Desired Solution

Instead of knowing very visually how much shielding and hull strength the target has with the target UI on the left hand side of the screen or with overlays on, this information would be vague to a player without a tactical scanner. This might mean that the player only knows whether a ship has its shields up or not and whether it is disabled or not. 

On the target UI, the player might see the shield count circle fade away with damage, or stay the same colour and disappear / fade away with very low shield hitpoint value. Hull value would fade from yellow to red with damage, but this fading would only happen when the ship is close to being disabled instead of fading away as soon as some hull damage is being done. This means that a player without a tactical scanner has less information in a battle and may do things like overestimate shield recharge / strength or hull strength, making tactical scanning potentially very valuable in these situations. 

With a tactical scanner, the player would be given the current UI where they can see how much shield / hull there is. 

### Alternative Approaches

Potentially, the tactical scanner could also do what the overlay does right now and bring up the shielding and hull of the current selected ship over the ship. I like using the overlay but it is not very nice looking and potentially confusing with all the overlays over each other in a battle with many enemies. Just having the one overlay for the ship I select would be nice. 

### Additional Context

<img width="139" alt="Normal" src="https://user-images.githubusercontent.com/115441627/208041572-6e2045d3-05ae-4401-ac8b-8506827451bb.png">

Current

<img width="139" alt="no shields low hull" src="https://user-images.githubusercontent.com/115441627/208041665-9a834578-548a-4d87-a4e3-c138207e0383.jpg">

Low shields, could be medium or high hull

<img width="139" alt="no shields low hull" src="https://user-images.githubusercontent.com/115441627/208042045-a9ecafcf-4e98-4276-80f6-16c183243117.png">

No shields, low hull

<img width="139" alt="no shields low hull" src="https://user-images.githubusercontent.com/115441627/208041611-d5b5ac70-7c37-4b6b-9056-18c5caaa2c09.png">

No shields, near disabled hull

**Comment by Zitchas** [comment]
I rather like this idea. The specifics may take some debate, but I've always found it odd that we know so much about the state of a ship right off the bat, and that tactical scanners (which *sound* like they should be very useful for everyone in battle) are virtually useless unless one is using heat/ion based combat. And even then it is marginal, since chances are most people using heat or ion already know if their target is susceptible or not, so it's just a matter of keep on shooting until they are disabled, tactical scanner just gives them an indicator to watch so they know their progress. Not exactly the peak of tactics, there.

**Comment by SimulatedKnave** [comment]
"How much damage have you done to this ship" and "how strong are its shields" may or may not be stuff that is possible to easily evaluate from visual or low-grade sensor inspection. 

If they are, the current implementation makes sense. If they're not, it doesn't.

**Comment by bene-dictator** [comment]
> "How much damage have you done to this ship" and "how strong are its shields" may or may not be stuff that is possible to easily evaluate from visual or low-grade sensor inspection.

In the game's dialogue, there's a few times where the player uses their scanner or can see visually whether ships' shields are up. I don't think there is anything in the story about seeing how powerful a ship's shield is by looking at it. I also like to think that ships especially not equipped for combat like the shuttle and the star barge have no point having combat functions like these. I want to keep a very minor visual indicator to let the player guess at the strength still, presumably through their bad scanners or visually.

**Comment by SimulatedKnave** [comment]
I mean...currently you don't actually know how much armour and shielding and such the ship actually has left. You just know what percentage it has left. Which DOES seem like the sort of thing you might be able to gauge visually, or with relatively simple sensors. 

Do tactical scanners tell you armour and hull and shield totals currently? If not, they should. And a little more vagueness might definitely be good. But the current system actually is pretty vague in the ways one might expect it to be.

**Comment by bene-dictator** [comment]
Total hull and shielding values would not be very useful and I don't know where they'd go on the UI. I'm trying to keep some visual indicators in case you're a combat ship and you don't want to sacrifice 4 outfit space for a scanner. That is the visual / bad scanners part. But the point is that ships that don't intend to be in combat shouldn't have information about shielding, a combat-related thing. 

> you don't actually know how much armour and shielding and such the ship actually has left. You just know what percentage it has left.

This doesn't make sense.

**Comment by SimulatedKnave** [comment]
When looking at the target display currently, you only know what percentage of the circle remains to be eliminated. While you can infer how much that is based on how much damage you know you do and how fast the circles disappear, a full health Bactrian and a full health Sparrow look the same on the targeting indicator. So do a half health Bactrian and a half health Sparrow. It's how fast the circle changes and from what that provides the actual information. Well, that and the player's knowledge. 

If all your shots are bouncing off the shields, even basic sensors can probably discern that. Exactly how strong the enemy's shields are is the sort of thing a more advanced scanner would be needed for. 

And shielding isn't just a combat-related thing. I'd assume it fills an important navigational role. Are there any actual ships with no shields at all (bar drones)?

**Comment by bene-dictator** [comment]
Again, I want to leave a small visual indicator to know vaguely how much damage you've done, which is the colour of the shield circle changing noticably but not as helpful in a battle. The point is that you're encouraged to get the tactical scanner to know how fast that circle disappears instead of looking at a vague fading circle. The colour of the circle could mean how much health is left, but with the scanner, you'll know the percentage. And using the tactical scanner, you'll be able to find out how strong the enemy's shields are. Shielding is also meant to be some sort of anti radiation thing to help keep your crew alive, but in game nothing happens if you don't have any shields. I don't know what the 'navigational role' means?

**Comment by SimulatedKnave** [comment]
By navigational role I mean the anti-radiation thing, and preventing micrometeoroids from flying through your ship from end to end. Flying around at high speed would logically be a lot riskier without shields.

**Comment by bene-dictator** [comment]
But what is your point?

**Comment by Amazinite** [comment]
I don't think that we should be nerfing normal gameplay in order to create (more of) a use for the tactical scanner.

> Just having the one overlay for the ship I select would be nice.

This makes more sense as a setting for the overlay. See #3911 as an example of this request.
